web/public/js/controllers/manage-event-applications-controller.js

```diff
@@ -322,6 +322,7 @@
       function updateDeleted() {
         application.updateAction = 'delete';
         application.deleted = true;
+        successMessage = $translate.instant('Ticket for') + ' ' + application.name + ' ' + $translate.instant('has been successfully deleted');
```

Comment:
I have just realised it might be better if we say it like:
Ticket for the following application has been successfully deleted: application.name"
Because otherwise this will be put up to the translation service in separate chunks.

Also, we will need to add the string [to the messages.po file here](https://github.com/CoderDojo/cp-zen-platform/blob/master/web/locale/en_US/messages.po).